apps/storybook-e2e/src/e2e/elements/form/vl-form-validation.stories-wc.cy.ts

```diff
@@ -0,0 +1,29 @@
+const formValidationUrl =
+    'http://localhost:8080/iframe.html?globals=backgrounds.value:!hex(F8F8F8)&viewMode=story&id=elements-form--form-validation';
+
+describe('story vl-form-validation - validation', () => {
+    it('should still validate when name of input is changed externally', () => {
+        cy.visit(`${formValidationUrl}`);
+
+        cy.get('[is="vl-form"]')
+            .find('[is="vl-input-field"]')
+            .each((inputFieldResult) => {
+                const inputField = inputFieldResult[0];
+                inputField.setAttribute('name', new Date().toString());
+            });
+
+        cy.get('[is="vl-form"]').find('[type="submit"]').click();
```

Comment:
als ik hier `cy.get('[is="vl-form"]').submit();` zet, dan slaagt de test lokaal, met flag `--watch` maar faalt dan op bamboo maar ook lokaal zonder `--watch` flag

vraag me of dit 2 verschillende cypress's zijn? in ieder geval op deze manier slaagt de test